devices/xiaomi/xiaomi_mho-c401n_temp_hum_sensor.json

```diff
@@ -0,0 +1,191 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "MiaMiaoCe",
+  "modelid": "MHO-C401N",
+  "vendor": "Xiaomi",
+  "product": "MHO-C401N",
```

Comment:
Nothing serious, but it would be good to use e.g. "Temperature/humidity sensor MHO-C401N". A bit more talkative :)